### ISSUE TITLE
Голосование на территории Крыма незаконно

### ISSUE THREAD

**Issue by DavertMik** [smartvotedocs/votesmart2021#49]
Согласно международному законодательству, которое, кстати исполняет GitHub, Крым является территорией Украины временно оккупированой РФ. 

![image](https://user-images.githubusercontent.com/220264/133616983-cc9cc87a-293b-4eb5-9e41-8d80eeb02907.png)

Потому любое голосование в госдуму на этой территори незаконное. 

---

Если вы сами поддерживаете беспредел, то почему удивляетесь беспределу по отношению к вам?

**Comment by dvantage** [comment]
> Согласно международному законодательству, которое, кстати исполняет GitHub, Крым является территорией Украины временно оккупированой РФ.
> 
> ![image](https://user-images.githubusercontent.com/220264/133616983-cc9cc87a-293b-4eb5-9e41-8d80eeb02907.png)
> 
> Потому любое голосование в госдуму на этой территори незаконное. Уберите из списка округа из Крыма.
> 
> Если вы сами поддерживаете беспредел, то почему удивляетесь беспределу по отношению к вам?

Нашёл к чему прикопаться)

**Comment by EgorkA82** [comment]
Да, но даже если крым и оккупирован (🤔), то на него, как на оккупированную территорию, распространяется Конституция РФ, основной закон государства. А выборы проводятся в соответстии с Конституцией

**Comment by schernichkin** [comment]
Даже сюда украинская свинья залезла.

**Comment by TheUnitedNat** [comment]
смею возразить, согласно Статье 79 Конституции РФ: "Российская Федерация может участвовать в межгосударственных объединениях и передавать им часть своих полномочий в соответствии с международными договорами Российской Федерации, если это не влечет за собой ограничения прав и свобод человека и гражданина и не противоречит основам конституционного строя Российской Федерации. Решения межгосударственных органов, принятые на основании положений международных договоров Российской Федерации в их истолковании, противоречащем Конституции Российской Федерации, не подлежат исполнению в Российской Федерации". Из этого следует, что Крым есть территория Российской Федерации, и любая компания, находящаяся на территории вынуждена это соблюдать.(жду того, как меня назовут кремлеботом, и напишут +15 руб.(но я не кремлебот, и призываю всех голосовать по УМГ))

**Comment by DavertMik** [comment]
@TheUnitedNat Можно выдумывать всё что угодно. Но есть нормы международного права, которое всегда будет выше законодательства РФ, что бы вы там себе не написали. Если вор и насильник напишет "меня могут судить только по законам Нарнии", это учитываться не будет. Его будут всё равно судить за воровство и изнасилование. Рано или поздно. И обпетлять вы всё равно не сможете. Первое же демократическое правительство, на которое вы уповаете будет договариваться о выходе с Крыма и Донбасса и на выплату контрибуции. 
Если вы хотите, чтобы к вам соблюдались общечеловеческие правила, начните с себя. Соблюдайте международные законы. Крым - это Украина, а не то что вы себе выдумали.

**Comment by andry81** [comment]
> Но есть нормы международного права

Международное право называется международным, потому-что его так назвали, а не потому-что все народы и государства готовы под ним подписаться или согласиться с ним. Это, уж извините, зависит от конкретных сиюминутных интересов народа или государства.

**Comment by TheUnitedNat** [comment]
@DavertMik о, уже увидел претензии, да, Крым - Украина, и я с этим согласен, но живу я в России, и тут международное право ниже конституции, и каждый кто здесь живёт вынужден с этим смириться, пока конституция такая. И про вора и насильника, если по законам Нарнии это не наказуемо, значит это не наказуемо

**Comment by DavertMik** [comment]
@TheUnitedNat Я могу понять вашу позицию. Но нельзя признавать законы, которые были приняты против правил. Если завтра вас заставят принести веревку и по закону заставят вешаться, вы признаете такой закон? Законодательство РФ касательно Крыма создает правовую коллизию, это правда. Но в Прекрасной России Будущего эта коллизия должна быть решена. И вам же лучше, если вы не будете лезть в эту ловушку Путина. Ведь любое следующее правительство увидит, что Крым это для РФ спорная территория (где не оперирует ни одна крупная компания, боясь санкций) и держать его на балансе это миллиарды в минус. Потому вариантов тут нет. Надо изначально отталкиваться от прстой позиции - Крым должен быть возвращен. Точка.

@andry81 
> Международное право называется международным, потому-что его так назвали, а не потому-что все народы и государства готовы под ним подписаться или согласиться с ним. Это, уж извините, зависит от конкретных сиюминутных интересов народа или государства.

Ну вот, согласно сиюминутных интересов некоторого государства желательно, чтоб вся оппозиция сидела в гулаге. И все сочувствующие там же. Ну и вы. А собственно, что вы тут делаете, раз не в гулаг хотите?

**Comment by andry81** [comment]
> Ну вот, согласно сиюминутных интересов некоторого государства желательно, чтоб вся оппозиция сидела в гулаге. И все сочувствующие там же. Ну и вы.

Странное заявление. Государство (текущая власть) хочет вовсе не этого. Ей нужно просто остаться у власти и продолжать остальных доить. Как-то глупо убивать овец несущих шерсть и мясо.

> А собственно, что вы тут делаете, раз не в гулаг хотите?

С чего Вы взяли, что я куда-то хочу? И причём тут гулаг?

**Comment by DavertMik** [comment]
@andry81 
> Странное заявление. Государство (текущая власть) хочет вовсе не этого. Ей нужно просто остаться у власти и продолжать остальных доить. Как-то глупо убивать овец несущих шерсть и мясо.

Так овцы несущие шерсть и мясо это не вы, а нефть и газ. А вы - баласт.

> С чего Вы взяли, что я куда-то хочу? И причём тут гулаг?

Всё просто. Если вы хотите жить по беспределу, то и с вами будут обходиться по беспределу. 
Сначала посадят навального, потом ФБК, потом всех кто зарегистрировался в его штабе, а потом всех кто отписался в этой теме. Для новой звездочки товарищу майору.

**Comment by andry81** [comment]
> Так овцы несущие шерсть и мясо это не вы, а нефть и газ. А вы - баласт.

Зачем тогда балласт налогами и поборами обкладывать?

> Всё просто. Если вы хотите жить по беспределу, то и с вами будут обходиться по беспределу.
> Сначала посадят навального, потом ФБК, потом всех кто зарегистрировался в его штабе, а потом всех кто отписался в этой теме. Для новой звездочки товарищу майору.

Ну а если не посадят, то что? С тебя взятки гладки?
Да и зачем сажать, достаточно припугнуть, они и сами свалят. Примеров уже полно.

**Comment by rmarinsky** [comment]
Да шо тут розмовляти, дітей пропаганди не переконати
Як ковтали так і будуть ковтати
Максимум може в снєжкі на вуличці побавунькаються
